packages/mendel-pipeline/package.json

```diff
@@ -22,10 +22,10 @@
     "cli-table": "^0.3.1",
     "commander": "^2.9.0",
     "debug": "^2.2.0",
-    "mendel-config": "^3.0.0-beta.2",
-    "mendel-deps": "3.0.0-beta.2",
```

Comment:
We are using beta version internally. But, this is with beta tag. New packages will be created with 2.0.0 stable tag.